torch/ao/quantization/pt2e/_numeric_debugger.py

```diff
@@ -71,6 +71,45 @@ def _assign_debug_handle(node: torch.fx.Node) -> None:
     bfs_trace_with_node_process(ep, _assign_debug_handle)
 
 
+def _detach(x):
```

Comment:
Add type annotations. `object` is fine if you do isinstance inside it and have a backup if it's not a tensor or a collection of tensors at all.

TypeVar might be helpful for this function

Reply:
I'm going to use Any